charts/k3s/templates/rbac/role.yaml

```diff
@@ -22,6 +22,9 @@ rules:
   - apiGroups: ["apps"]
     resources: ["statefulsets", "replicasets", "deployments"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: ["policy"]
```

Comment:
Should be moved to .Values.rbac.role.extended below

Reply:
done